geotrek/api/v2/serializers.py

```diff
@@ -333,6 +333,52 @@ class Meta:
         fields = ('id', 'advice', 'filter', 'name', 'pictogram')
 
 
+class HDViewPointSerializer(serializers.ModelSerializer):
+    geometry = geo_serializers.GeometryField(read_only=True, source="geom_transformed", precision=7)
+    picture_tiles_url = serializers.SerializerMethodField()
+    thumbnail_url = serializers.SerializerMethodField()
+    trek = serializers.SerializerMethodField()
+    site = serializers.SerializerMethodField()
+    poi = serializers.SerializerMethodField()
+    license = serializers.SlugRelatedField(
+        read_only=True,
+        slug_field='label'
+    )
+    create_datetime = serializers.DateTimeField(source='date_insert')
```

Comment:
Not related to your issue directly but : it would be a good idea to add a mixin for create_datetime and update_datetime as it is use in multiple places in api/v2, wouldn' it ?